src/metabase/pulse/render/body.clj

```diff
@@ -335,7 +335,9 @@
       :labels      labels}
      (when (:graph.show_goal viz-settings)
        {:goal {:value (:graph.goal_value viz-settings)
-               :label (or (:graph.goal_label viz-settings) (tru "Goal"))}}))))
+               :label (or (:graph.goal_label viz-settings) (tru "Goal"))}})
+     (when (:series_settings viz-settings)
+       {:series_settings (:series_settings viz-settings)}))))
```

Comment:
This additional information is needed to allow series with custom colors to display properly. These custom colors will take precedence over the default color.

https://github.com/metabase/metabase/pull/26211/files#diff-e7653fc1bf6052860248c9be6e3726efda50058c3108c819329e9689c9d87c44R86

![image](https://user-images.githubusercontent.com/1937582/199998018-41ec187e-5b86-41cd-8ea3-c0a95bddd64e.png)

Reply:
I think eventually the combo chart implementation should follow those in Guage and Row charts where BE does as little work as possible and let FE do its own things.

The reason being that:
1. Now BE kind of reimplements some logic that already exists in the FE
2. It's a bit cumbersome at the moment to extend features/fix bugs on certain static charts since BE limited information that would be sent to FE, whereas more information are present for app static viz, so to achieve feature parity we need to keep passing more and more information, so we might as well pass the same information to FE static viz altogether.
3. Doing it this way makes static viz and app viz APIs stand apart, and also makes it harder to achieve feature parity and share code.

Not that I'm saying we shouldn't clean up this function. But depending on future priority, if we're to rework the combo chart maybe we don't need to clean up this function at all (since it'll be reimplemented anyway).

Reply:
+1 to @WiNloSt, let's follow the pattern we have with `Gauge` and `Row` chart when we have time to implement it

Reply:
Yea, +1 here too. Makes good sense :)